composer.json

```diff
@@ -19,16 +19,22 @@
     }
   ],
   "require": {
-    "php": ">=7.1",
+    "php": "^7.2",
+    "contributte/di": "dev-next",
```

Comment:
Must be changed before merge
https://github.com/contributte/di/pull/15

Reply:
contributte/di merged

Reply:
Just forgot to push. Done